tests/Aspire.Hosting.Azure.Tests/ExistingAzureResourceTests.cs

```diff
@@ -1254,6 +1275,314 @@ param keyVaultName string
             }
             """;
 
+        output.WriteLine(BicepText);
```

Comment:
Is this just here to help with test output?

Reply:
It's there to help when the bicep changes and you need to know what changed. You can copy the actual bicep from the test log and paste it into the test code. That will show you all the differences. This is better/easier than getting the diffs one at a time from the test failure:

![image](https://github.com/user-attachments/assets/78370b10-aa47-440f-ad67-1ec673478fde)